libs/wire-api/src/Wire/API/User/Client/Prekey.hs

```diff
@@ -70,9 +69,8 @@ instance ToSchema Prekey where
 clientIdFromPrekey :: Prekey -> ClientId
 clientIdFromPrekey =
   ClientId
-    . decodeUtf8
-    . toByteString'
-    . encodeBase16
+    . foldl' (\w d -> (w `shiftL` 8) .|. fromIntegral d) 0
```

Comment:
it would be nice if there was documentation on this.

Reply:
I added a haddock.